app/src/main/java/com/example/infinite_track/presentation/geofencing/GeofenceManager.kt

```
@@ -376,8 +382,8 @@ class GeofenceManager @Inject constructor(
         return if (location.locationId != 0) {
             "active:$activeAttendanceId:${location.locationId}"
         } else {
-            val lat = String.format("%.6f", location.latitude)
-            val lng = String.format("%.6f", location.longitude)
+            val lat = String.format("%.6f", location.coordinate.latitude)
```

detekt (1.23.8)

[warning] String.format("%.6f", location.coordinate.latitude) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)